gke/setup_ssl.sh

```diff
@@ -19,4 +19,4 @@ DOMAIN=$(yq eval '.domain' config.yaml)
 echo $PROJECT_ID
 gcloud config set project $PROJECT_ID
 
-gcloud compute ssl-certificates create website-certificate --domains=$DOMAIN --global
\ No newline at end of file
+gcloud compute ssl-certificates create dc-website-cert --domains=$DOMAIN --global
```

Comment:
Same as above-what's the intent of changing to `dc-website-cert` ?

Reply:
Same as above, helm expects a cert named `dc-website-cert` instead of `website` here: https://github.com/datacommonsorg/website/blob/84eea6a8a5521e7eede559835d8a06c3ddab85c2/deploy/helm_charts/dc_website/values.yaml#L46